sources/core/Stride.Core.Mathematics.Tests/TestMatrix.cs

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) .NET Foundation and Contributors (https://dotnetfoundation.org/ & https://stride3d.net) and Silicon Studio Corp. (https://www.siliconstudio.co.jp)
```

Comment:
When adding new code to Stride (in a new file) don't include Silicon Studio Corp in the copyright notice.